README.md

```diff
@@ -47,11 +49,26 @@ Project | Aras
 8. Click **Import** in the top left corner.
 9. Optional: Repeat steps 5-8 for `..\GenericActionItem\Import\2-Import - Example on DCO (optional)\1-Import\imports (admin).mf`.
 10. Optional: Repeat steps 5-8 for `..\GenericActionItem\Import\2-Import - Example on DCO (optional)\2-Import\imports (admin).mf`.
-11. Close the Aras Package Import tool.
+11. Optional: Repeat steps 5-8 for `..\GenericActionItem\Import\3-Import - Example Action Item Template (optional)\imports (admin).mf`.
+12. Close the Aras Package Import tool.
 
 ## Usage
-
-See [generic Action Item Concepts v1-2(A10).pdf](./Documentation/generic%20Action%20Item%20Concepts%20v1-2(A10).pdf) for more information on using this project.
+This package can be used one of two ways:
+* by use of an Action Item Template set up by an Administrator
+* Added to an existing DCO item by a user either as ad-hoc or by using an existing template as above
+
+> This item may also be configured as a relationship to other items, but must be set to a Create New type of relationship.
+
+###Example Usage: 
+1. (Optional) Create Template(s) for action items
+2. Create New or Open Existing Express DCO
+  * if new, must save item before using template action
+3. Use Action to add from template or add ad hoc using tab
```

Comment:
Try to be very specific about how people need to do things. Which action? How do they run that action? Is it a toolbar button, main menu action, or context menu action?